Dockerfile

```
@@ -0,0 +1,32 @@
+FROM python:3.7-slim-buster as base
+RUN apt-get update \
+&& apt-get install -y \
+build-essential \
+libssl-dev \
+zlib1g-dev \
+libbz2-dev \
+libreadline-dev \
+libsqlite3-dev \
+wget \
+curl \
+llvm \
+libncurses5-dev \
+libncursesw5-dev \
+xz-utils \
+tk-dev \
+libffi-dev \
+liblzma-dev \
+python-openssl \
+git \
+gunicorn \
+&& curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
```

You don't need most of these
```suggestion
RUN apt-get update \
&& apt-get install -y curl \
&& curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
```